### ISSUE TITLE
Reproducing visualisation in paper

### ISSUE THREAD

**Issue by jeff024** [deeplearning-wisc/cider#7]
Hi, could you please provide code for reproducing Figure 3 in your paper on UMAP visualisation? If not, could you please help me by identifying a few details for reproducing this Figure?

1. Did you used training set or validation set of CIFAR-10 for visualisation?
2. What are hyper-parameters used for UMAP visualisation? Such as n_neighbours, min_dist etc. ?

Many thanks in advance

**Comment by ElonJustin7** [comment]
Hi, author. I have a question regarding Figure 3 in your paper on UMAP visualization. Is the visualization based on the embeddings produced by the projection head? I'm curious about how the embedding from the projection head is obtained for the left side where cross-entropy (CE) is applied. In your code, the SupCEHeadResNet consists of an encoder and classifier, without including a projection head. Could you please clarify this for me? Thank you！
